google/cloud/spanner_v1/transaction.py

```diff
@@ -191,18 +191,19 @@ def execute_update(self, dml, params=None, param_types=None, query_mode=None):
         transaction = self._make_txn_selector()
         api = database.spanner_api
 
+        seqno = self._execute_sql_count
+        self._execute_sql_count += 1
```

Comment:
Multiple assignment would be atomic here, e.g.:
```suggestion
        seqno, self._execute_sql_count = self._execute_sql_count, self._execute_sql_count + 1
```